helpers/uuid_test.go

```diff
@@ -0,0 +1,101 @@
+package helpers_test
```

Comment:
why you test wrappers?

Reply:
as blackbox, we do not care what we use inside

Reply:
for what?

Reply:
it is easy to change the internal implementation

Reply:
For what?